ci/jenkins/test.sh

```diff
@@ -461,6 +470,9 @@ function deliver_antrea {
     if [[ $FLEXIBLE_IPAM == true ]]; then
         control_plane_ip="$(kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 ~ role {print $6}')"
         scp -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" build/yamls/*.yml jenkins@${control_plane_ip}:~
+    elif [[ $CONTAINERD_JUMPER == true ]]; then
+        control_plane_ip="$(kubectl get nodes -o wide --no-headers=true | awk -v role="control-plane" '$3 == role {print $6}')"
```

Comment:
Is it better to use "$CONTROL_PLANE_NODE_ROLE" here instead of "control-plane"? Maybe we should also change the CONTROL_PLANE_NODE_ROLE value in line 45 to "control-plane."